### DISCUSSION TITLE
How to pair tasmotized Zigbee bridge, remotes and bulbs correctly

### DISCUSSION THREAD

**Discussion by Th0maz** [arendst/Tasmota#18568] [Q&A] [answered]
Hi,
I need some help with the way of pairing my zigbee devices correctly.
I've got a Sonoff zigbee bridge runnung tasmota 12.2.0 and some IKEA Tradfri bulbs and remotes. How can I pair all the devices correctly, so they understand each other?

What I tried:
- reset remote
- allowed pairing in bridge and remote got paired. I could see remote buttons in the console when pressed.
- reset bulb
- allowed pairing in bridge again and bulb got paired. I could control the bulb with zbsend commands.
- put remote close to bulb and pressed end held pairing button for 10 seconds
- remote can control bulb. I can still control the bulb by zbsend commands as well.
But:
- I no longer get updates from the bulbs in tasmota/bridge, when controlled by remote (not bridge)
- I no longer see the remote commands in tasmota console, no battery status update as well.

Is that supposed to work or did I understand the whole zigbee stuff wrong?

**Comment by Jason2866** [comment]
Have you tried pairing the Remote to the Tasmota Zigbee Bridge?
You should see incoming messages from the Remote. You have to do the logical connection from the Remote to control "something (bulb)" yourself!

**Reply by Th0maz** [reply]
Yes, that's what I did in the first and second step.
But after pairing the remote to the bulbs as well, the connection of the remote to the bridge gets lost. I wasn't sure if this is even possible to keep it connected to both bridge and bulbs.
I would prefer the remote to controll the bulbs directly, without using any rules or event broker by myself. If this is not possible, I will keep it the way it is now, with a heavy heart.

**Reply by Jason2866** [reply]
Not possible. Either bulb or Tasmota zigbee coordinator.

**Reply by Th0maz** [reply]
ok, that's not the answer I wanted to hear 😩
Thanks! 😉

**Comment by Th0maz** [comment]
To clarify what I had in mind:
I would like to
- switch on lights "directly", even if bridge/network/mqtt/server becomes unavailable
- use the spare buttons on the remote to control "something else" via the bridge, because my bulbs can't change light color anyway

**Reply by s-hadinger** [reply]
You can do this with binding. But only for switches that support it. For example an ikea switch can be bound to a bulb or a group of bulbs even if the coordinator is down

**Reply by Th0maz** [reply]
This sounds exactly like what I wanted to do.
Will try this out, thanks!

**Reply by Th0maz** [reply]
I tried this and re-paired all devices to the bridge (after ZBForget for every device).
After that I could see the remote button presses in the console again.
I used ZBSend to assign a group to the bulbs and ZBBind to bind the remote to this group as described in the [tasmota documentation](https://tasmota.github.io/docs/Zigbee/#zigbee-binding).
After that I could control the bulbs again with my remote, even when the tasmota zigbee bridge was unavailable/restarting.
BUT:
I am not able to see the remote button presses anymore after binding the remote to the bulbs. And the current state of the bulbs is only shown correctly in tasmota, when sent via the bridge, not when switched by the remote.
This looks like the behaviour I already had before 🤔

**Reply by Th0maz** [reply]
Not sure if this is of any help to understand what happened here.
The documentation says about Zigbee Binding:
"The coordinator will still receive all commands."
![grafik](https://user-images.githubusercontent.com/12065628/235949813-edbb9b64-bcee-409a-a146-838b6e63a474.png)

**Reply by Th0maz** [reply, marked as answer]
I added
`ZBListen1 210` (which is my device group) 
and now I see the status of my bulbs in tasmota and receive the remote button presses 🥳

I think I have to run this command on every (re-)boot of the bridge. have to find a good trigger for the rule ;-)
